src/main/java/com/univocity/parsers/common/ArgumentUtils.java

```diff
@@ -156,6 +156,18 @@ public static int indexOf(char[] array, char element, int from) {
 		return -1;
 	}
 
+
+	/**
+	 * Public interface for indexOf for testing purposes
+	 * @param array
+	 * @param element
+	 * @param from
+	 * @return
+	 */
+	public static int publicIndexOf(Object[] array, Object element, int from){
```

Comment:
I don't really like this way of testing, but if that is how it's supposed to be, sure :)(: